site/blog/2016/jul/1/sphinx-and-rtd-for-writers.rst

```diff
@@ -0,0 +1,548 @@
+Generally it's easiest to output HTML for local writing and testing,
+and then you can let Read the Docs generate PDF's and other formats.
+
+This article will provide an overview of the features of Sphinx and Read the Docs,
+and allow you to evaluate them for use in your organization.
+
+Introduction to Sphinx
+----------------------
+
+`Sphinx <http://www.sphinx-doc.org/>`_ is the documentation tool of choice for the Python language community,
+and increasingly large parts of the software ecosystem.
+It was `originally created <https://en.wikipedia.org/wiki/Sphinx_(documentation_generator)>`_ in 2008 to document the Python language itself.
+
+Over the past eight years,
+it has turned into a robust and mature solution for software documentation.
+It has a number of features that technical writers would expect,
+including:
+
+* Single Source Publishing 
+    * Output HTML, PDF, ePub, and more
+    * Content reuse through includes
+    * Conditional includes based on content type and tags
+* Multiple mature HTML themes that look beautiful on mobile and desktop
+* Referencing across pages, documents, and projects
+* Index and Glossary support
+* Internationalization support
+
+It also has practical and powerful tools for documenting software specifically,
+including:
+
+* Semantic referencing of software concepts, including Classes, Functions, Programs, Variables, etc.
+* Including of code comments inside prose documentation for many programming languages
+* Tools for documenting HTTP API's
+* Extensions with the Python language
+* A vast array of third party extensions to do useful things
+
+This article isn't large enough to cover all of the power packed into this tool.
+Hopefully I can show you enough that it piques your interest and you can research more.
+
+Using Sphinx
+~~~~~~~~~~~~
+
+reStructuredText is a powerful language primarily because the syntax can be extended.
+There are two types of extension that reStructuredText supports:
+
+* Paragraph level (with Directives)
+* Inline level (with Interpreted Text Roles, or roles for short)
+
+Paragraph Level Markup
+``````````````````````
+
+Let's see a basic example of a *Directive* in reStructuredText::
+
+    .. warning:: Here be dragons! This topic covers a number of options that
+       might alter your database.
+
+       Proceed with caution!
+
+    .. figure:: screenshot-control-panel.jpg
+       :width: 50%
+
+       An overview of the admin control panel.
+
+Here ``warning`` acts as the name of the directive,
+and is the part you can extend for custom directives.
+In the ``figure``,
+``screenshot-control-panel.jpg`` is an *argument*,
+``:width:`` is an *option*,
+and the rest is the *content*
+They allow for customization of directives,
+and show the full power of reStructuredText.
+
+.. note:: You'll notice that Sphinx uses whitespace to denote where a directive ends.
+   This means the content will last until the next unindented line.
```

Comment:
Uh -- Cannot tell exactly what this is supposed to mean "This means the content lasts ..." but I *think* it's supposed to mean that the markup of the content as a [what type of presentation element?] is closed when the whitespace is added. Is that right?